.github/workflows/release.yml

```diff
@@ -19,12 +23,24 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: "3.10"
+      
+      - name: Prepare Package Version
+        run: |
+          sed -i 's/__PACKAGE_VERSION__/1.34.2/g' zanshincli/version.py
+          sed -i 's/version = "0.1.0"/version = "1.34.2"/g' pyproject.toml
+# sed -i 's/__PACKAGE_VERSION__/${{ github.event.release.tag_name }}/g' zanshincli/version.py
```

Comment:
This is the code you want, but it's commented. Seems like you committed the local code in the way you set it up for testing, but I can't approve the PR like this.

Reply:
> This is the code you want, but it's commented. Seems like you committed the local code in the way you set it up for testing, but I can't approve the PR like this.

I missed the push button haha, now it's uploaded.